openmetadata-ui/src/main/resources/ui/src/components/MyData/FeedWidget/FeedWidget.component.tsx

```diff
@@ -73,7 +77,7 @@ const MyFeedWidgetInternal = ({
       undefined,
       undefined,
       undefined,
-      10
+      PAGE_SIZE_MEDIUM
```

Comment:
confirm this, here you are using PAGE_SIZE_MEDIUM and below in this file you are using PAGE_SIZE_BASE.
Why it is different ?

Reply:
We are fetching 25 responses at a time and displaying 15 in a widget. If the widget has more than 15 responses, we show the 'View More' button so users can view all the data. Ideally, we should fetch data without a limit rather than 25, but if we do not provide a limit, it takes 10 as a default value.